src/config.rs

```diff
@@ -122,6 +122,7 @@ create_config! {
     struct_trailing_comma: SeparatorTactic,
     struct_lit_trailing_comma: SeparatorTactic,
     struct_lit_style: StructLitStyle,
+    struct_lit_force_multiline: bool,
```

Comment:
Could you make an enum for this rather than using a bool please? Otherwise in the future when we add more options then people who use this will have their configs broken.